kernel/packages/decentraland-ecs/types/@decentraland/EnvironmentAPI/index.d.ts

```diff
@@ -5,6 +5,13 @@ declare module '@decentraland/EnvironmentAPI' {
     serverName: string
     displayName: string
   }
+
+  export type ExplorerData = {
+    clientUrl: string
+    buildNumber: number
+    configurations: Record<string, any>
```

Comment:
Could it more specific with the Record<string, any> to limit the use case to Record<string, string>? ala process.env

that way we ensure type safety for the present usages and a safeguard for the future

Reply:
I'll make a compromise: Strings, number or booleans.

Reply:
good enough